numpy/_core/src/multiarray/descriptor.c

```diff
@@ -1409,7 +1409,8 @@ PyArray_DescrConverter2(PyObject *obj, PyArray_Descr **at)
  * TODO: This function should eventually receive a deprecation warning and
  *       be removed.
  *
- * @param descr
+ * @param descr descriptor to be checked
+ * @param DType pointer to the Dtype of the descriptor
```

Comment:
```suggestion
 * @param DType pointer to the DType of the descriptor
```
I would prefer to try to use the `DType` capitalization consistently also in flowing text.  (in part because that is what composite words in german would do maybe, but mainly, I think it is clearer to use the code-name where even before me, `DataType` is far more common than `Datatype` in NumPy.)